### DISCUSSION TITLE
Error "Publii cannot find your sites folder"

### DISCUSSION THREAD

**Discussion by zipswich** [GetPublii/Publii#1662] [Help]
I moved the folder, so this error is expected.
However, Publii quits after this message without giving me a chance to correct it.  
Could anyone offer a tip on how to remedy this?

**Comment by bjazmoore** [comment]
Put it back.  Back it up with the backup function or copy and paste it somewhere.  Delete the site using Publii's site "remove" function.  This will insure that there are no left over settings that Publii cannot resolve.

**Comment by zipswich** [comment]
I restored the folder, then deleted the site within Publii. 
Now, it allows me to create a website but does not allow me to open one from a folder.

**Comment by zipswich** [comment]
Finally figured out a way to do it but I do not know if that is the right way.
The original file location:
 E:\Publii
Change it in app settings to the new location:
 C:\Publii

The original site is under E:\Publii\my-space
Delete the site.
Create a new site My Space. Publii creates folder 
 C:\Publii\my-space
Close Publii.
Copy all the folders under the original folder my-space to the new folder my-space.
Open Publii and the new site with all the original posts are there.
 
Everything appears to be restored including website sync information. I was afraid that I would lose it. I do not know where the sync information is stored.
